package.json

```diff
@@ -7,8 +7,8 @@
     "test": "test"
   },
   "scripts": {
-    "test": "grunt test",
-    "build": "bower install; grunt build"
+    "build": "bower install; grunt build",
+    "test": "./node_modules/loopback-testing/node_modules/.bin/mocha common/models/test"
```

Comment:
`npm test` should run both server and client tests. Please keep `grunt test` and rework the Gruntfile: rename the task `test` to `test:client`, add a `test:server` task to run Mocha tests in `server/test`, and add a `test` task to run `test:server` and `test:client`.